src/plugins/drawing/model/drawing-content.test.ts

```diff
@@ -38,7 +38,10 @@ describe('defaultDrawingContent', () => {
   });
   it('should return content with optional stamps', () => {
     const stamps = [{ url: "my/stamp/url", width: 10, height: 10 }];
-    const appConfig = AppConfigModel.create({ config: { stamps } as any });
+    const appConfig = AppConfigModel.create({
+      config: { stamps } as any,
+      "curriculumBaseUrl": "http://localhost:8080",
```

Comment:
Is this URL actually used? I'm guessing not since `my/stamp/url` seems fake. If it isn't used then its probably better to use something like `https://exmaple.com`

Reply:
Thanks for catching that. I've changed the URL.